neo/sys/glimp.cpp

```diff
@@ -249,7 +249,11 @@ bool GLimp_Init(glimpParms_t parms) {
 		// try to put the window on the display the mousecursor currently is on
 		{
 			int x, y;
+#if SDL_VERSION_ATLEAST(2, 0, 4)
 			SDL_GetGlobalMouseState(&x, &y);
+#else
```

Comment:
you could put that whole loop that's trying to guess  the `displayIndex` and the definition of `int x, y;` into the `#if SDL_VERSION_ATLEAST(2, 0, 4)` block, and in the else case just set `displayIndex = 0;`